mackup/config.py

```diff
@@ -206,7 +207,11 @@ def _parse_path(self):
                 raise ConfigError("The required 'path' can't be found while"
                                   " the 'file_system' engine is used.")
 
-        return str(path)
+        # Python 2 and python 3 byte strings are different.
+        if sys.version_info[0] < 3:
+            return str(path)
+        else:
+            return path.decode("utf-8")
```

Comment:
Yeah, even now a string is being returned. Previously this was not working. This might make things a bit more clearer of how byte strings are handled in python3.

```
>>> var = b'abc'
>>> var
b'abc'
>>> str(var)
"b'abc'"
>>> var.decode("utf-8")
'abc'
```